core/templates/dev/head/services/CsrfTokenServiceSpec.ts

```diff
@@ -24,54 +24,39 @@ import $ from 'jquery';
 // TODO(#7222): Remove the following block of unnnecessary imports once
 // the code corresponding to the spec is upgraded to Angular 8.
 import { UpgradedServices } from 'services/UpgradedServices';
+import {CsrfTokenService} from './CsrfTokenService';
 // ^^^ This block is to be removed.
 
 require('services/CsrfTokenService.ts');
 
 describe('Csrf Token Service', function() {
-  var $httpBackend = null;
-  var $q = null;
-  var $rootScope = null;
-  var CsrfTokenService = null;
-
-  beforeEach(angular.mock.module('oppia'));
-  beforeEach(angular.mock.module('oppia', function($provide) {
-    var ugs = new UpgradedServices();
-    for (let [key, value] of Object.entries(ugs.upgradedServices)) {
-      $provide.value(key, value);
-    }
-  }));
-  beforeEach(angular.mock.inject(function(
-      _$httpBackend_, _$q_, _$rootScope_, _CsrfTokenService_) {
-    $httpBackend = _$httpBackend_;
-    $q = _$q_;
-    $rootScope = _$rootScope_;
-    CsrfTokenService = _CsrfTokenService_;
-
-    this.scope = $rootScope.$new();
-
-    spyOn($, 'ajax').and.returnValue($q.resolve({token: 'sample-csrf-token'}));
-  }));
+  let csrfTokenService: CsrfTokenService ;
+  // let $q: any = new Promise((resolve, reject) => {
+  //   resolve({token: 'sample-csrf-token'});
+  // });
+
+  beforeEach(() => {
+    csrfTokenService = new CsrfTokenService();
+    // spyOn($, 'ajax').and.returnValue($q);
```

Comment:
@Showtim3 any reason for this?
I think we should still spy on $.ajax since that is what is used to send the ajax request.